packages/client/src/v2-events/features/events/useCertificateTemplateSelectorFieldConfig.ts

```diff
@@ -9,14 +9,35 @@
  * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
  */
 
-import { FieldConfig, FieldType } from '@opencrvs/commons/client'
+import formatISO from 'date-fns/formatISO'
+import {
+  areCertificateConditionsMet,
+  EventDocument,
+  EventState,
+  FieldConfig,
+  FieldType
+} from '@opencrvs/commons/client'
 import { useAppConfig } from '@client/v2-events/hooks/useAppConfig'
 
 export const CERT_TEMPLATE_ID = 'certificateTemplateId'
 export const useCertificateTemplateSelectorFieldConfig = (
-  eventType: string
+  eventType: string,
+  declaration: EventState,
+  event?: EventDocument
```

Comment:
When do we pass the event and when we not need to?

Reply:
Good point. Not sure why it was initially implemented like that.
We should always pass it in. I've made the change to fix that.